app/src/main/java/com/example/mycalculator/mvp/view/CalculatorView.java

```diff
@@ -17,5 +19,19 @@ public CalculatorView(MainActivity mainActivity, ActivityMainBinding binding) {
     public void drawNumber(String number) {
         binding.screenTextExample.setText(number);
     }
+    @Override
+    public void showErrorDivision(String errorMessage) {
```

Comment:
you are not using the parameter variable, remove it pls

Reply:
Done

Reply:
this still here